Automattic/wp-calypso
# Pattern Assembler - Enable unified-pages/virtual-home-page on Horizon

#### Proposed Changes

* Enable unified-pages/virtual-home-page on Horizon for testing

#### Testing Instructions


* Access your site on horizon.wordpress.com
* Go to "Settings > Readings", and set "Your homepage displays" to "Your latest posts"
* Go to "Pages", and you will see the virtual page, Home, listed in “Published” tabs
* Search the page by "Home", and you will see the Home page is still there
* Click on "...", and you're able to edit, view or copy the link of the Home page


#### Pre-merge Checklist





Related to https://github.com/Automattic/wp-calypso/issues/69011, https://github.com/Automattic/wp-calypso/issues/69158, https://github.com/Automattic/wp-calypso/issues/69171
